test/e2e/runner/setup.go

```diff
@@ -209,6 +209,7 @@ func MakeGenesis(testnet *e2e.Testnet) (types.GenesisDoc, error) {
 	}
 	genesis.ConsensusParams.Evidence.MaxAgeNumBlocks = e2e.EvidenceAgeHeight
 	genesis.ConsensusParams.Evidence.MaxAgeDuration = e2e.EvidenceAgeTime
+	genesis.ConsensusParams.ABCI.VoteExtensionsEnableHeight = testnet.InitialHeight + 10
```

Comment:
This causes the chain to _update_ to using vote extensions. I think this is somewhat nice to have in an e2e test because it can help catch cases of issue with the handover. 

Another option would be to add this as a param into the `testnet` toml files so it can vary with each test. Thoughts?

Reply:
I thought we decided allowing the network to vary on this setting was a non-goal, in which case is it not sufficient to vary the enable height?

(We can simulate "never" by setting it to 0 I think? And "always" with 1?)

Reply:
Ah, perhaps unclear: That's what I mean. Have it vary between tests, so we can have it enable at different points in the future. Not have it vary within a test.

Reply:
Oh I see, that makes more sense than what I was thinking. I agree it probably does make sense since we plan to allow networks not to enable it at all.

Reply:
Yeah, I think best balance is what you have now. If we need more flexibility we can walk the extra mile and put it into the testnet toml, but I don't think the extra effort is worth it now.